service/src/main/java/bio/terra/policy/service/pao/graph/model/AttributeEvaluator.java

```diff
@@ -0,0 +1,140 @@
+package bio.terra.policy.service.pao.graph.model;
+
+import bio.terra.policy.common.model.PolicyInput;
+import bio.terra.policy.service.pao.model.Pao;
+import bio.terra.policy.service.policy.PolicyMutator;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This code does the processing that turns a group of policy attributes from PAOs into a single,
+ * effective attribute set. The expected usage is: 1. construct the evaluator 2. add the attributes
```

Comment:
nit: might be easier to read if numbered items were on new lines.

Reply:
Rewrote into a proper javadoc form with html tags